src/main/java/org/spongepowered/api/plugin/PluginManager.java

```diff
@@ -45,6 +48,17 @@
     Optional<PluginContainer> fromInstance(Object instance);
 
     /**
+     * Finds the {@link PluginContainer} for the specified plugin instance.
+     *
+     * @param instance Plugin instance
+     * @return Plugin container of instance
+     * @throws PluginNotFoundException if the instance could not be found
+     */
+    default PluginContainer find(Object instance) {
```

Comment:
`fromInstance(Object)` returns an optional. Normally you could just call `get` if you assume it is present and it will also throw an exception otherwise.